tests/ensemble_evaluator/conftest.py

```diff
@@ -0,0 +1,142 @@
+import json
+import os
+from pathlib import Path
+from unittest.mock import Mock
+
+import pytest
+from ert_shared.ensemble_evaluator.entity.ensemble import (
+    create_ensemble_builder,
+    create_legacy_job_builder,
+    create_legacy_stage_builder,
+    create_realization_builder,
+    create_step_builder,
+)
+from res.enkf import ConfigKeys
+from res.enkf.queue_config import QueueConfig
+from res.job_queue.driver import LOCAL_DRIVER
+from res.job_queue.ext_job import ExtJob
+
+
+@pytest.fixture
+def queue_config():
+    return QueueConfig(
+        config_dict={
+            ConfigKeys.JOB_SCRIPT: "job_dispatch.py",
+            ConfigKeys.USER_MODE: True,
+            ConfigKeys.MAX_SUBMIT: 100,
+            ConfigKeys.NUM_CPU: 10,
+            ConfigKeys.QUEUE_SYSTEM: LOCAL_DRIVER,
+            ConfigKeys.QUEUE_OPTION: [
+                {ConfigKeys.NAME: "MAX_RUNNING", ConfigKeys.VALUE: "50"}
+            ],
+        }
+    )
+
+
+@pytest.fixture
+def make_ensemble_builder(queue_config):
+    def _make_ensemble_builder(tmpdir, num_reals, num_jobs):
+        builder = create_ensemble_builder()
+        with tmpdir.as_cwd():
+            ext_job_list = []
+            for job_index in range(0, num_jobs):
+                ext_job_config = Path(tmpdir) / f"EXT_JOB_{job_index}"
+                with open(ext_job_config, "w") as f:
+                    f.write(f"EXECUTABLE ext_{job_index}.py\n")
+
+                ext_job_exec = Path(tmpdir) / f"ext_{job_index}.py"
+                with open(ext_job_exec, "w") as f:
+                    f.write(
+                        f"""#!/usr/bin/env python
```

Comment:
```suggestion
                        f"#!/usr/bin/env python"
                        f"if __name__ == '__main__':"
                        f"    print('stdout from {job_index}')"
```

The formatting makes me itch